src/Features/Core/Portable/CodeRefactorings/AddMissingImports/IAddMissingImportsFeatureService.cs

```diff
@@ -25,17 +26,17 @@ internal interface IAddMissingImportsFeatureService : ILanguageService
         /// if there are ambiguous imports, no known resolutions to import, or if no imports that would be provided
         /// would be added without adding a reference for the project. 
         /// </summary>
-        Task<Document> AddMissingImportsAsync(Document document, TextSpan textSpan, AddMissingImportsOptions options, CancellationToken cancellationToken);
+        Task<Document> AddMissingImportsAsync(Document document, TextSpan textSpan, AddMissingImportsOptions options, IProgress<CodeAnalysisProgress> progressTracker, CancellationToken cancellationToken);
 
         /// <summary>
         /// Analyzes the document inside the texstpan to determine if imports can be added.
         /// </summary>
         Task<AddMissingImportsAnalysisResult> AnalyzeAsync(Document document, TextSpan textSpan, AddMissingImportsOptions options, CancellationToken cancellationToken);
 
         /// <summary>
-        /// Performs the same action as <see cref="AddMissingImportsAsync(Document, TextSpan, AddMissingImportsOptions, CancellationToken)"/> but
+        /// Performs the same action as <see cref="AddMissingImportsAsync(Document, AddMissingImportsAnalysisResult, SyntaxFormattingOptions, IProgress{CodeAnalysisProgress}, CancellationToken)"/> but
```

Comment:
>AddMissingImportsAnalysisResult, SyntaxFormattingOptions [](http://example.com/codeflow?start=84&length=56)

I'm guessing these were intentionally changed, but I'm not clear on why.

Reply:
just threading through the progress object because we *do* have it.

Reply:
Am I reading this wrong, it sounds like this is saying it performs the same action as the same method, whereas before it was saying it did the same as the prior method.